unirec/Makefile.am

```diff
@@ -79,8 +79,8 @@ endif
 
 deb-clean:
 	rm -rf unirec_*.build unirec_*.changes unirec_*.deb unirec-dev_*.deb unirec_*.orig.tar.gz unirec-*.tar.gz unirec-@VERSION@
-
+if !DX_COND_doc
```

Comment:
This solution is not good.
Cleaning fields.* and ur_values.* files should not be dependent on doxygen.
Remove `doxygen-clean` dependency and move existing `clean-local` outside the `DX_COND_doc condition (and delete the else branch completely).
Then append `rpm-clean` and `deb-clean` as a dependencies of `clean-local` target.